auto_submit/lib/service/revert_review_template.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class RevertReviewTemplate {
+  RevertReviewTemplate({
+      required this.repositorySlug,
+      required this.revertPrNumber, 
+      required this.revertPrAuthor,
+      required this.originalPrLink,}) {
+    constructTitle();
+    constructBody();
+  }
+
+  String repositorySlug;
+  int revertPrNumber;
+  String revertPrAuthor;
+  String originalPrLink;
+
+  String? _title;
+  String? _body; 
+
+  void constructTitle() {
```

Comment:
1. Should these be private?
2. Since these have a verb like construct, should they return the string instead?

Reply:
That makes sense. I'll go ahead and change the method to return the string and also make the methods private.